packages/backend/src/services/SavedChartsService/SavedChartService.ts

```diff
@@ -685,10 +685,23 @@ export class SavedChartService {
                 "You don't have access to the space this chart belongs to",
             );
         }
+        const sinceDaysDiff = 60;
+        const versions = await this.savedChartModel.getLatestVersionSummaries(
+            chartUuid,
+            sinceDaysDiff,
+        );
+        analytics.track({
+            event: 'saved_chart_history.view',
+            userId: user.userUuid,
+            properties: {
+                projectId: chart.projectUuid,
+                savedQueryId: chart.uuid,
+                versionCount: versions.length,
+                sinceDaysDiff,
```

Comment:
why would we want to send this since days diff 60?

You want to send this probably? `moment().subtract(sinceDaysDiff, 'days'),` -> from `getLatestVersionSummaries`